gradlew.bat

```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off
```

Comment:
Try to keep the commit simple. There should be no need to change the `gradlew.bat` file. Please revert the change.

Reply:
Done!